### PR TITLE
Add CascadeUploader part at IPFS Upload Section

### PR DESCRIPTION
**Pull request by mastercodercat**
Updated IPFS Upload Section regarding to new plugin - data-provider-cascade.

### CODE REVIEW

File: docs/learn/expert-guides/utilize-ipfs-storage.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ If you want to set up your own workflow for uploading and retrieving files using
 - [`tools-data-providers`](https://github.com/lukso-network/tools-data-providers)
 - [`service-ipfs-proxy`](https://github.com/lukso-network/service-ipfs-proxy)
 
-To use IPFS as a file service through _Pinata_ and _Infura_, you will have to:
+To use IPFS as a file service through _Pinata_, _Infura_ and _Csacade_ you will have to:
```
Comment (Hugoo) [inline docs/learn/expert-guides/utilize-ipfs-storage.md:16]:
```suggestion
To use IPFS as a file service through _Pinata_, _Infura_ and _Cascade_ you will have to:
```

**Review by Hugoo — APPROVED** [review]
Cool thanks!

File: docs/learn/expert-guides/utilize-ipfs-storage.md [hunk 2]
```diff
@@ -84,6 +84,14 @@ const url = await provider.upload(file);
 console.log('File URL:', url);
 ```
 
+**Using Cascade**
+
+```js
+import { CascadeUploader } from '@lukso/data-provider-cascade';
```
Comment (Hugoo) [inline docs/learn/expert-guides/utilize-ipfs-storage.md:90]: I can't find that lib on NPM?

https://www.npmjs.com/search?q=%40lukso%2Fdata-provider-cascade

**Review by Hugoo — CHANGES_REQUESTED** [review]
Adding the RC. See questions in the reviews.

File: docs/learn/expert-guides/utilize-ipfs-storage.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ If you want to set up your own workflow for uploading and retrieving files using
...
-3. **Upload your File Content**: Utilize the [LUKSO network tools for data providers](https://github.com/lukso-network/tools-data-providers) to upload content. You can upload directly to Pinata using your Pinata credentials or to the proxy with the shared secret.
+1. **Setup your Gateway Account**: Register at [Pinata](https://www.pinata.cloud/), [Infura](https://www.infura.io/) and [Cascade](https://cascade.pastel.network) and ensure the IPFS gateway is enabled on your Infura account. This will grant you access to their service endpoints.
+2. **Configure your Proxy**: Deploy a proxy on Cloudflare using secrets from Infura, Pinata, Cascade and a shared secret of your choice. This setup allows for a customized Pinata gateway for uploads and enables downloads via a subscription.
+3. **Upload your File Content**: Utilize the [LUKSO network tools for data providers](https://github.com/lukso-network/tools-data-providers) to upload content. You can upload directly to Pinata using your Pinata credentials or to the proxy with the shared secret. And also by using cascade api key, you can upload to Cascade protocol directly.
```
Comment (Hugoo) [inline docs/learn/expert-guides/utilize-ipfs-storage.md:20]:
```suggestion
3. **Upload your File Content**: Utilize the [LUKSO network tools for data providers](https://github.com/lukso-network/tools-data-providers) to upload content. You can upload directly to Pinata using your Pinata credentials or to the proxy with the shared secret. And also by using Cascade API key, you can upload to Cascade protocol directly.
```

File: docs/learn/expert-guides/utilize-ipfs-storage.md (hunk 2 above)
Comment (Hugoo) [reply docs/learn/expert-guides/utilize-ipfs-storage.md:90]: Related to: https://github.com/lukso-network/tools-data-providers/pull/12

**Comment by Hugoo** [comment]
https://github.com/lukso-network/tools-data-providers/pull/12

**Comment by Hugoo** [comment]
Let's merge this on the lib has been published?

**Comment by Hugoo** [comment]
Waiting for https://github.com/lukso-network/tools-data-providers/pull/12 to be merged and released - @richtera

**Review by CJ42 — CHANGES_REQUESTED** [review]
We should put this on hold until we have the sub-package from the tool-data-provider released. @richtera

**Comment by CJ42** [comment]
@mastercodercat the CI is failing and there are merge conflicts. Could you fix them please so that we can merge? :)

**Pull request closed without merging**